brigand/react-style-proptype
# Add test folder to npmignore to prevent it from being bundled in release

## Context
Right now, when `react-style-proptype` is included in a project that uses flow and does not ignore the `react-style-proptype` node module, Flow throws an error because of the `flow-not-ok.js` test.

This was introduced by the `v3.2.0` release and while I looked into it, I found out that the `v3.1.0` release did not include the test folders and the `v3.2.0` release does.

I question whether the `flow-not-ok.js` test is at all relevant (because basically you are testing Flow functionality here) I think that it is a good practice that test folders that test the functionality of this module on itself should not be included in an npm package

## What has been done?
- I added `test` to `.npmignore` to prevent the `test` folder from being published in the NPM Package

## How to test?
- Publish a new npm version
- See that tests are not included
